libs/mark-flow-ui/src/components/candidate_contest.tsx

```diff
@@ -179,21 +206,22 @@ export function CandidateContest({
     }
 
     updateVote(contest.id, newVote);
-    setRecentlyDeselectedCandidate(id);
+    setRecentlyDeselectedCandidate(candidate);
   }
 
-  function handleUpdateSelection(candidateId: string) {
+  function handleUpdateSelection(candidate: Candidate) {
     /* istanbul ignore else - @preserve */
-    if (candidateId) {
-      const candidate = findCandidateById(vote, candidateId);
-      if (candidate) {
-        if (candidate.isWriteIn) {
-          setWriteInPendingRemoval(candidate);
+    if (candidate) {
```

Comment:
Seems like it's not possible for `candidate` to be undefined here, so this `if` is unnecessary now